.github/workflows/integration-tests-vm-type.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     env:
       COLLECTOR_IMAGE: quay.io/rhacs-eng/collector:${{ inputs.collector-tag }}
       COLLECTOR_QA_TAG: ${{ inputs.collector-qa-tag }}
-      TEST_IMAGE: quay.io/rhacs-eng/qa-multi-arch:collector-tests-${{ inputs.collector-tag }}
+      TEST_IMAGE: quay.io/rhacs-eng/qa-multi-arch:collector-tests-3.18.0
```

Comment:
Do we need to roll this change back? I see `main.yml` and `nightly-tests.yml` are passing `3.18.0-slim` as the version used to `integration-tests.yml` and that should forward it to this workflow.

Reply:
This is a bit trickier, because `inputs.collector-tag` contains the `-slim` suffix, which is not how we tag the test image. I think there's a couple of approaches we can use: 
- add the -slim suffix to the collector image here and revert to using `inputs.collector-tag` for the test image
- we can add a new input to provide the test image tag directly (which I've actually done [in the other PR](https://github.com/stackrox/collector/pull/1537/files#diff-f2221670add2a7da88201d6683873ebbc6571df15a32a427edf160b4c2c29cca) so it might be nice for consistency)

WDYT?